client/state/states-list/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getStatesList( state, countryCode ) {
```

Comment:
The `-list` suffix is a convention that was used in emitter object stores, but you'll see most directories in `client/state` don't use it. Instead, you might consider simplifying to `client/state/states` and revising the selectors to reflect this: `getStates( state, countryCode )` or `getCountryStates( state, countryCode )`